pkgs/data/fonts/alegreya-sans/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, fetchzip }:
+
+fetchzip {
+  name = "alegreya-sans";
+
+  url = "https://www.huertatipografica.com/free_download/23";
+
+  postFetch = ''
+    mkdir -p $out/share/fonts
+    unzip -j $downloadedFile 'AlegreyaSans*.otf' -d $out/share/fonts/opentype
+  '';
+
+  sha256 = "07yi9viil2k4hrw18525cc9apcz9ynk1vsinc5n7snwnky2mj97y";
+
+  meta = with lib; {
+    description = "Humanist sans serif family with a calligraphic feeling";
+    longDescription = ''
+      Alegreya Sans is a humanist sans serif family with a calligraphic feeling that conveys a dynamic and varied rhythm, This gives a pleasant feeling to readers of long texts.
```

Comment:
Typo, there should be a `.` instead of a `,` before `This`.

This `longDescription` seems very long indeed, and the long lines are hard to review. Maybe break them up? But I have no idea how this would affect rendering.

Reply:
I'm not sure the longDescriptions are particularly informative to be honest. I suspect the original author was mainly trying to use more characters to show off their font rather than inform the reader. I can't comment on the rendering, as I'm not even sure where they are used (`nix search` doesn't seem to return them in any of its output formats).

Reply:
I don't know if they are somewhere in `nix search`, but the longDescriptions are shown on the [packages site](https://nixos.org/nixos/packages.html?channel=nixpkgs-unstable&query=frescobaldi). I still sometimes look for packages there, as that's the way I was shown when I first started using nixos :D

Reply:
Hmm. That looks like a good example of terrible rendering 😮: Prose, rendered in a fixed width font, with hard line breaks, but also word wrapping which wraps the last few words of each line on my screen (dependent on window and font sizes).

For some reason, [this mahoosive longDescription](https://github.com/NixOS/nixpkgs/blob/af3c1000a4c519348a527a07497646c915a01857/pkgs/data/fonts/oxygenfonts/default.nix#L20) appears as "Not specified" on the [packages site](https://nixos.org/nixos/packages.html?channel=nixpkgs-unstable&query=oxygenfonts).

Reply:
True. Just not using monospace would go a long way I guess. Maybe open an issue or PR over at [nixos-homepage](https://github.com/NixOS/nixos-homepage)?

Reply:
[Issue](https://github.com/NixOS/nixos-homepage/issues/473) opened... and subsequently closed, as they will soon be switching to a new page which renders it better.

I also fixed the typo.